### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by ronys** [pwsafe/pwsafe#1891]
(Received via email from Kristof)

Following are several accessibility issues. Sub-issues (new tickets marking this as the parent) should be created for each as they are addressed.

> Hi and best wishes;  I am a blind user of Password Safe (for Windows).  LIKE IT A LOT!  But going from 3.65 to 3.72 I find some issues:
> 
> 1) (highest pri), The Alt+N kbd-seq to get to the 'Notes' field in the 'Basic' tab of an entry has gone.  Don't need it, but want it (PLEASE)! (or a suitable replacement, of course!)
> 
> 2) I hope and trust you will think about kbd-seqs to access the new features (RFC-6238 Authenticator, 'Copy Password' button, 'Custom Fields' etc).  For the most part PwSafe is VERY accessible for people who don't use a pointing device, please think about this.  Good to have keystrokes to flip between Notes and Custom Fields, for instance.  Also please think about what gets focus after such a keystroke.  The 'Notes' and 'Custom Fields' sub-tabs is intuitively nice, but nifty key-stroke navigation of this would make PwSafe even more special!
> 
> Indeed, I note that kbd shortcuts (default ones, that is) seem to have gone from the context menu when an entry is selected in the list/tree entries views.  E.g. Ctrl+U to copy user name, Ctrl+G to copy note content ...;  perhaps you are working on this?  And, indeed, this could explain some of my later comments below.
> 
> Also the Esc key no longer works when the Note content is in focus;  deliberately so?
> 
> 3) (a) I note that, using arrow keys to scroll through the list of custom fields, Ctrl+C does not copy a field's value because entry is not selected when arrowing, nor with spacebar.  As a screen reader user, I have to use the screen reader's mouse emulation to select the entry by emulating a "click" in the right place, and this is awkward.  (In fact I have just added a second Custom Field and find that arrowing when the list is in focus does not work at all!)
> (The 'Copy Custom Field value' function on the entry context menu in the standard views works fine without mouse use.)
> 
> 3) (b) Neither does Ctrl+C work any longer when scrolling through the password entries in turn;  When I press Ctrl+C when an entry is selected(?) to copy the password, my screen reader complains: "Nothing Selected".  (It actually says this to me).  in 3.65 and all previous V3s Ctrl+C works.  (I now realise the default shortcut Ctrl+C is also removed from the context menu.  In preparation for introducing a new set of default shortcuts?)
> 
> 4) The Tab strip in the 'Preferences' dialog never receives focus when just using the Tab key to navigate through the forms in that dialog.  So I can navigate between tabs only with Ctrl+Tab and Ctrl+Shift+Tab.  (Also applies to V3.65.)  I have to confess I didn't regularly upgrade PwSafe for maybe half a decade, so I don't know when this issue came about.
> 
> 5) (A long-standing issue since I started using PwSafe which must be for over 1½ decades);  Since I navigate only with kbd shortcuts and the Tab key, I find that focus is "trapped" in the places where I can enter a shortcut key.  Once I tab-navigate into such a field, I'm jammed there!  Not even the Esc key works.  Once again I have to resort to mouse emulation to "escape" this situation.  Consequently I have never tried to take advantage of this feature.  (A lot of blind people would latch on to this facility and make heavy use of it if it worked properly, I think.)
> 
> 6) The initial open screen for PwSafe is a bit hostile for people who do not use a pointing device.  Focus starts off in the 'Master Password' field.  If I want to check which vault is applicable, I Shift+Tab twice to find a list of recent files.  If I then press Down-Arrow, a new file is highlighted, but focus immediately jumps back to the 'Master Password' field.  So if I want to make sure of the name of the newly selected vault, I have to do Shift+Tab twice again for my screen reader to read out the name of the vault.  What would be helpful is if that focus jump did not happen and/or if their were keystrokes available for the recent list and the master password field.
>
